CMakeLists.txt

```diff
@@ -25,6 +25,9 @@ set(CMAKE_CXX_STANDARD_REQUIRED TRUE)
 ecbuild_add_option( FEATURE OMP
                     DESCRIPTION "support for OpenMP shared memory parallelism"
                     REQUIRED_PACKAGES "OpenMP COMPONENTS Fortran" )
+if( CMAKE_Fortran_COMPILER_ID MATCHES "Cray")
+  add_link_options("-homp")
+endif()
```

Comment:
This should not be needed or needs to be implemented using the OpenMP::OpenMP_Fortran target depending on the content of HAVE_OMP variable. I thought this was already working.